components/operator-service/src/main/java/com/wso2telco/dep/operatorservice/model/APIBlacklistWhitelistResponseDTO.java

```diff
@@ -0,0 +1,34 @@
+package com.wso2telco.dep.operatorservice.model;
+
+public class APIBlacklistWhitelistResponseDTO {
+    boolean validationError;
```

Comment:
redundant variable? Need to manipulate value depending on another parameter. Better to use evaluation in isValidationError(). Single line execution is worth manipulation somewhere down the line again and again.